Guides/MAX78000 Motor Monitoring Case Study with SampleMotorDataLimerick Dataset.md

```diff
@@ -0,0 +1,146 @@
+# Motor Fault Detection at the Edge using MAX78000
+
+## A Case Study with the SampleMotorDataLimerick Dataset
+
+
+
+## Overview
+
+Condition-based monitoring (CbM) is the automated monitoring of equipment conditions. This process includes collecting sensor data and applying algorithms or machine learning techniques to predict the health status of the equipment. The equipment of interest is usually motors used in many industrial applications.
+
+The fact that the unhealthy, a.k.a. faulty data types, are hard to collect with varying types of operations leads to the use of “unsupervised” CbM models. Hence, only healthy data are used for model training and several fault types are tested for detection performance.
+
+The following sections will present the fault detection system for the edge devices. The designed model is trained, deployed, and verified on MAX78000 EVkit. Model performance is verified and evaluated using in-house sample motor data available from [https://github.com/analogdevicesinc/CbM-Datasets](https://github.com/analogdevicesinc/CbM-Datasets).
+
+
+
+## Dataset Information
+
+### Raw Data Properties
+
+Sample motor data is collected using a SpectraQuest Machinery Fault Simulator. The setup looks as follows:
+
+![SampleMotorDataLimerick - Overview](resources/SampleMotorDataLimerick_generalSetup.png)
+
+Faults tested include a variety of mechanical faults as follows:
+
+- Very Light Imbalance
+- Light Imbalance
+- Heavy Imbalance
+- Very Heavy Imbalance
+- Bent Shaft
+- Misaligned Shaft (Angular)
+- Light Inner Race Fault
+- Heavy Inner Race Fault
+- Light Outer Race Fault
+- Heavy Outer Race Fault
+- Light Ball Bearing Fault
+- Heavy Ball Bearing Fault
+
+
+
+For imbalance faults, varying loads are applied in the setup, a sample can be seen here:
+
+<img src="resources/SampleMotorDataLimerick_sampleImbalance.png" alt="Sample Imbalance" style="zoom:33%;" />
+
+
+
+Details for other fault types:
+<img src="resources/SampleMotorDataLimerick_otherFaults.png" alt="Other Faults" style="zoom: 33%;" />
+
+
+
+Each fault was tested at 600, 1200, 1800, 2400 & 3000 RPM and tested both with and without an additional 11 lb (5 kg) load on the shaft. ADXL356 sensor data is used for vibration raw data.
+
+For ADXL356 sensor, the sampling frequency was **20 kHz** and data csv files recorded for **2 seconds** in the X, Y and Z directions.
+
+Data files are saved in `.csv` format with the following naming convention:

```

Comment:
I'd suggest to add an example of a csv file format here.

Reply:
* Sure, I think we can also link an actual csv file in our Datasets folder after it is published in GitHub
* For Limerick dataset it is a little more complicated: healthy identifier is the string '_GoB_GS_BaLo_WA_' in the file name wrt the file naming convention, reflecting a:  Good Bearing, Good Shaft, Balanced Load and Well Aligned